packages/prompts/README.md

```diff
@@ -156,3 +156,32 @@ const group = await p.group(
 
 console.log(group.name, group.age, group.color);
 ```
+
+### Building
+
+Just like `group`, but on `builder` way, so you can choose which one fits better.
+
+```js
+import * as p from '@clack/prompts';
+
+const results = await p
+  .builder()
+  .add('name', () => p.text({ message: 'What is your name?' }))
+  .add('age', () => p.text({ message: 'What is your age?' }))
+  .add('color', ({ results }) =>
+    p.multiselect({
+      message: `What is your favorite color ${results.name}?`,
+      options: [
+        { value: 'red', label: 'Red' },
+        { value: 'green', label: 'Green' },
+        { value: 'blue', label: 'Blue' },
+      ],
+    })
+  )
+  .onCancel(() => {
```

Comment:
I am wondering if it be good to add exitCode as the second param to get rid of the `process.exit(0)` or adding it within `p.cancel('Canceled', 0)`. What are your thoughts ?